src/features/favorites/components/__tests__/FavoritesResults.test.tsx

```diff
@@ -13,29 +17,32 @@ jest.mock('features/favorites/pages/FavoritesWrapper', () => ({
   }),
 }))
 
+jest.mock('features/favorites/pages/useFavorites')
+const mockUseFavorites = useFavorites as jest.Mock
```

Comment:
Je vais faire comme si je n'avais pas vu 🙈  :
```suggestion
const mockUseFavorites = useFavorites as jest.MockedFunction<...>
```

Reply:
Oui pardon faut s'y habituer, c'est OK